vm/vmgs/vmgstool/src/main.rs

```diff
@@ -331,8 +331,9 @@ fn parse_legacy_args() -> Vec<String> {
 }
 
 fn main() {
-    DefaultPool::run_with(async |_| {
-        if let Err(e) = do_main().await {
+    DefaultPool::run_with(async |_| match do_main().await {
+        Ok(_) => eprintln!("The operation completed successfully."),
```

Comment:
Dude, what is this, DOS?

Reply:
My general preference, and I think the modern practice, is for CLI tools to say nothing when they do what is asked, perhaps unless the user specifies `--verbose` or something.

But I don't feel strongly here--I never use this tool.

Reply:
Alright.